libs/sdk-ui-theme-provider/src/ThemeProvider/Context.tsx

```diff
@@ -4,10 +4,16 @@ import compose from "lodash/flowRight";
 import { ITheme } from "@gooddata/sdk-backend-spi";
 import { wrapDisplayName } from "@gooddata/sdk-ui";
 
-const ThemeContext = React.createContext<ITheme | undefined>(undefined);
+/**
+ * @beta
+ */
+export const ThemeContext = React.createContext<ITheme | undefined>(undefined);
```

Comment:
Do we really need to export this? I mean for use in class components you could wrap them in the already present HOCs, right? Or am I missing something?